src/corecel/random/distribution/detail/GenerateCanonical32.hh

```diff
@@ -17,38 +17,29 @@ namespace detail
  * Generate random numbers in [0, 1) from a 32-bit generator.
  */
 template<class RealType = ::celeritas::real_type>
-class GenerateCanonical32;
+struct GenerateCanonical32;
 
 template<>
-class GenerateCanonical32<float>
+struct GenerateCanonical32<float>
 {
-  public:
-    //!@{
-    //! \name Type aliases
     using result_type = float;
-    //!@}
 
-  public:
-    //! Sample a random number with floating point precision
+    // Sample a random number with floating point precision
     template<class Generator>
-    inline CELER_FUNCTION result_type operator()(Generator& rng);
+    inline CELER_FUNCTION float operator()(Generator& rng);
```

Comment:
What's the reason for this change?

Reply:
Trying to fix warnings in the doxygen documentation (I can revert those)